Team417_2021/src/main/java/org/firstinspires/ftc/team417_2021/ginisTeleOp.java

```diff
@@ -0,0 +1,142 @@
+    public CRServo wristServo;
+    public DcMotor motorFL;
+    public DcMotor motorBL;
+    public DcMotor motorFR;
+    public DcMotor motorBR;
+
+    boolean toggleState = true;
+
+    public DcMotor motorArm1;
+    public DcMotor motorArm2;
+
+    public DcMotor motorSlides;
+
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+
+        motorFL = hardwareMap.dcMotor.get("motorFL");
+        motorBL = hardwareMap.dcMotor.get("motorBL");
+        motorFR = hardwareMap.dcMotor.get("motorFR");
+        motorBR = hardwareMap.dcMotor.get("motorBR");
+
+        wristServo = hardwareMap.crservo.get("wristServo");
+        wristServo.resetDeviceConfigurationForOpMode();
+        wristServo.setPower(0);
+
+        motorArm1 = hardwareMap.dcMotor.get("arm1");
+        motorArm2 = hardwareMap.dcMotor.get("arm2");
+
+        motorSlides = hardwareMap.dcMotor.get("motorSlides");
+
+        motorFL.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorBL.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorFR.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorBR.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorArm1.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorArm2.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorSlides.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+
+
+         //motorArm1.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.FLOAT);
+        motorArm1.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+        motorArm2.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+        motorSlides.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+
+        motorFL.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorBL.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorFR.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorBR.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorArm1.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorArm2.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorSlides.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+
+        motorFL.setDirection(DcMotorSimple.Direction.REVERSE);
+        motorBL.setDirection(DcMotorSimple.Direction.REVERSE);
+        motorArm1.setDirection(DcMotorSimple.Direction.FORWARD);
+        motorArm2.setDirection(DcMotorSimple.Direction.FORWARD);
+
+        motorFL.setPower(0);
+        motorBL.setPower(0);
+        motorFR.setPower(0);
+        motorBR.setPower(0);
+
+        motorArm1.setPower(0);
+        motorArm2.setPower(0);
+
+        motorSlides.setPower(0);
+
+        telemetry.addLine("initialized");
+        telemetry.update();
+
+        //indicates init has been hit?
+        waitForStart();
+
+        while(opModeIsActive()) {
+
+            toggleState = wristToggle.toggle(gamepad2.a);
+
+            if(toggleState) {
+                wristServo.setPower(0.7);
+            }
+            else {
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1-braces
```suggestion
            } else {
```